test/core/core_test.cpp

```diff
@@ -0,0 +1,81 @@
+#include <gtest/gtest.h>
+#include "core/core.cpp"
+
+    const std::string jsonInput = R"(
+        {
+            "receipt": [
+                {
+                    "products": [
+                        {
+                            "product": "Item1",
+                            "amount": 2,
+                            "price": 10.5,
+                            "whoBought": {
+                                "personID": 1
+                            }
+                        },
+                        {
+                            "product": "Item2",
+                            "amount": 1,
+                            "price": 5.99,
+                            "whoBought": {
+                                "personID": 2
+                            }
+                        }
+                    ]
+                }
+            ],
+            "users": [
+                {
+                    "id": 1,
+                    "name": "Wladek Mandziuk",
+                    "phone": "123-456-7890"
+                },
+                {
+                    "id": 2,
+                    "name": "Michal Jakistam",
+                    "phone": "987-654-3210"
+                }
+            ]
+        }
+    )";
+
+TEST(ParseJsonDataTest, ParsesJsonDataCorrectly) 
```

Comment:
What does mean "correctly"? Should rather be sth like "getUsersShouldReturnUsers"